### PR TITLE
FCM react native guide

### PR DESCRIPTION
**Pull request by sumitsaurabh927**

### CODE REVIEW

File: guides/novu-fcm-rna.mdx [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+  const token = await messaging().getToken();
+  // Log the FCM token
+  console.log('FCM Token is:', token);
+
+  // Return a function to unsubscribe from notifications
+  return unsubscribe;
+} else {
+  // If permission is not granted, log the error status
+  console.log('Could not grab token...', authStatus);
+}
```
Comment (unicodeveloper) [inline guides/novu-fcm-rna.mdx:60]: Is there anything that needs to be imported here? Where does `messaging()` come from?

**Review by unicodeveloper — COMMENTED** [review]
Nice work! Please attend to the comments @sumitsaurabh927

File: guides/novu-fcm-rna.mdx [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+```
+
+## FCM handlers
+FCM uses something called a `background handler` to, as the name suggests handle notifications when the app is in the background. So, we'll add it to our app:
+``` jsx
+// Background message handler for incoming messages while the app is in bg
+messaging().setBackgroundMessageHandler(async (remoteMessage) => {
+  // Log a message indicating that a message was handled in the background
+  console.log('Message handled in the background!', remoteMessage);
+});
```
[the remark this answers is not in the document]
Comment (sumitsaurabh927) [reply guides/novu-fcm-rna.mdx:71]: It is inside a conditional statement and is executed every time it evaluates to `true`. It is not meant to be assigned to any variable as it is a method of `messaging()`. A new instance of messaging is created inside the conditional on which `setBackgroundMessageHandler` is called

**Comment by sumitsaurabh927** [comment]
Pushed a new commit with fixes @unicodeveloper

**Review by unicodeveloper — APPROVED** [review]

**Pull request merged**